src/core/lib/executors/Node.ts

```diff
@@ -782,16 +782,20 @@ export default class Node extends Executor<NodeEvents, Config, NodePlugins> {
       delete config.suites;
 
       if (!require.extensions['.ts']) {
-        if (
-          ((config.node &&
-            config.node.suites.some(pattern => pattern.endsWith('.ts'))) ||
-            (config.plugins &&
-              config.plugins.some(plugin => plugin.script.endsWith('.ts')))) &&
-          typeof this.config.node.tsconfig === 'undefined'
-        ) {
-          register();
-        } else if (this.config.node.tsconfig) {
+        if (this.config.node.tsconfig) {
```

Comment:
Minor consistency nit -- is there a reason we're using `this.config` here, but just `config` below?